tests/unit/sagemaker/serializers/jumpstart/test_serializers.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+
+from mock.mock import patch
+
+from sagemaker import base_serializers, serializers
+from sagemaker.jumpstart.utils import verify_model_region_and_return_specs
+
+from tests.unit.sagemaker.jumpstart.utils import get_special_model_spec
+
+
```

Comment:
please unit test both methods: `retrieve_options` and `retrieve_default`

same for `deserializers`

Reply:
don't i currently do that? maybe i should break into multiple test cases for clarity